plugin/PDFView.qml

```diff
@@ -190,7 +198,41 @@ SilicaFlickable {
 
     children: [
         HorizontalScrollDecorator { color: Theme.highlightDimmerColor },
-        VerticalScrollDecorator { color: Theme.highlightDimmerColor }
+        VerticalScrollDecorator { color: Theme.highlightDimmerColor },
+        PDFContextMenu {
+            id: contextMenuLinks
+            //% "External link"
+            title: qsTrId("sailfish-office-tl-pdf-link")
+            MenuItem {
+                text: contextMenuLinks.message.indexOf("mailto:") === 0 ?
+                //% "Open mail composer"
```

Comment:
You're right, I'll change it for this (specific case for http(s), and generic for others).